package.json

```diff
@@ -46,7 +46,8 @@
     "istanbul": "0.4.2",
     "mocha": "2.4.5",
     "nyc": "7.0.0",
-    "semantic-release": "^4.3.5"
+    "semantic-release": "^4.3.5",
+    "sinon": "1.17.4"
```

Comment:
I've never used `sinon` before. Can you please explain in what cases is it useful? Btw, I'm going out right now but I'll review this a little further in a few hours.